app/src/main/java/com/github/leuludyha/ibdb/presentation/screen/auth/signup/SignUpScreenViewModel.kt

```diff
@@ -17,30 +19,55 @@ class SignUpScreenViewModel @Inject constructor(
 
     private var walkThroughCompleted by mutableStateOf(false)
 
+    //TODO: All methods here are implemented using SharedPreferences. I believe this will be changed
+    // further in development, as Dylan intended to implement userPreferences in a different way.
+    // This is just a temporary way of doing it.
+
```

Comment:
Actually, I meant to use `DataStore` which is the newer equivalent of shared preferences. I don't think it really matters which one we use as long as it works.